dashboard/src/app/dashboard/[dashboardId]/layout.tsx

```diff
@@ -56,6 +57,13 @@ export default async function DashboardLayout({ children, params }: DashboardLay
                 <UsageUpgradeBanner billingDataPromise={getUserBillingData()} />
               </Suspense>
             )}
+            {isFeatureEnabled('enableAccountVerification') &&
+              session.user?.email &&
+              !session.user?.emailVerified && (
```

Comment:
Regarding the hook from the other comment, it would be nice if this could be simplified to:
```suggestion
            {userEmail.verified === false && (
```

If the `isFeatureEnabled('enableAccountVerification')` is true, then `verified` could also just return true for instance